rest/blip_client_test.go

```diff
@@ -514,30 +626,63 @@ func NewBlipTesterClientOptsWithRT(tb testing.TB, rt *RestTester, opts *BlipTest
 	if err != nil {
 		return nil, err
 	}
+
 	client.pullReplication.bt.avoidRestTesterClose = true
 	client.pushReplication.bt.avoidRestTesterClose = true
 
 	return client, nil
 }
 
+func (btc *BlipTesterClient) Close() {
+	for _, collectionReplicator := range btc.CollectionClients {
+		collectionReplicator.Close()
+	}
+}
+
+func (btc *BlipTesterClient) initCollectionReplication(collection string, collectionIdx int) error {
+	btcReplicator := &BlipTesterCollectionClient{
+		docs:                 make(map[string]map[string]*BodyMessagePair),
+		attachments:          make(map[string][]byte),
+		lastReplicatedRev:    make(map[string]string),
+		BlipTesterClientOpts: &btc.BlipTesterClientOpts,
+		pullReplication:      btc.pullReplication,
+		pushReplication:      btc.pushReplication,
+		rt:                   btc.rt,
+	}
+
+	btcReplicator.collection = collection
+	btcReplicator.collectionIdx = collectionIdx
+
+	btc.CollectionClients[collection] = btcReplicator
+	return nil
+}
+
+// DummyCollection() just returns the BlipTesterCollectionClient with empty key. This is used when not running in a
+// collection mode
+func (btc *BlipTesterClient) DummyCollection() *BlipTesterCollectionClient {
```

Comment:
dummy -> default
remove implementation detail 

```suggestion
// DefaultCollection returns the client's default collection
func (btc *BlipTesterClient) DefaultCollection() *BlipTesterCollectionClient {
```

Reply:
@bbrks I had Default originally but changed it. I didn't want it to be confused with the actual `_default` collection.... Are you fine with this?

Reply:
I renamed it back but kinda apprehensive. It does kinda use _default collection but doesn't actually use any of the getCollection stuff so kinda weird.